docs/USING_ADVANCED.md

```diff
@@ -41,18 +41,18 @@ console.log(myMarked('I am using __markdown__.'));
 
 |Member      |Type      |Notes                                                                                                                        |
 |:-----------|:---------|:----------------------------------------------------------------------------------------------------------------------------|
-|baseUrl     |`??`      |Default is `null`                                                                                                            |  
+|baseUrl     |`string`  |A prefix url for any relative link.  Default is `null`               |  
```

Comment:
If we add the columns, think the order would be:

|Member |Type |Default |Added in |Notes|

Reply:
Thanks! I went with `Since` and put `v0.4.0` so its clear that its clear that this is a version number. I couldn't easily find when these other features were introduced so I used `???`